pkg/lib/cockpit.js

```diff
@@ -4190,10 +4190,10 @@
         self.get = function get(path, params, headers) {
             return self.request({
                 method: "GET",
-                params: params,
-                path: path,
+                params,
+                path,
                 body: "",
-                headers: headers
+                headers
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-192102-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)